app/src/main/res/layout-xhdpi/control_main_list_item.xml

```diff
@@ -98,6 +108,16 @@
                 android:gravity="center"
                 android:layout_gravity="center" />
 
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:layout_marginTop="5dp"
+                android:layout_gravity="center"
+                android:text="Hz"
+                android:textSize="12sp"
+                android:paddingTop="9dp"/>
+
+
```

Comment:
Try to run the "Reformat Code" option in Android studio so it will automatically take off these extra spaces and indent the code correctly